karma.conf.js

```diff
@@ -319,7 +319,7 @@ module.exports = function (config) {
 			{
 				pattern:
 					config.grep ||
-					'{debug,devtools,hooks,compat,test-utils,jsx-runtime,}/test/{browser,shared}/**/*.test.js',
+					'{debug,devtools,hooks,test-utils,jsx-runtime,}/test/{browser,shared}/**/*.test.js',
```

Comment:
Will undo before checking in. Did this temporarily locally to reduce testing noise as I tackled failures in the core tests.